dialect/sql/schema/postgres.go

```diff
@@ -294,16 +294,38 @@ func (d *Postgres) scanColumn(c *Column, rows *sql.Rows) error {
 		}
 		c.SchemaType = map[string]string{dialect.Postgres: udt.String}
 	}
+	return scanPostgresColumnDefault(c, defaults)
+}
+
+// scanPostgresColumnDefault used to scan defaults for postgres column
+func scanPostgresColumnDefault(c *Column, defaults sql.NullString) (err error) {
```

Comment:
What do you think about skipping the error in case of failure? The current migration doesn't support migrating default values atm, so it has no effect.

Reply:
how about just track the raw default without the Scan when read from db ?

there are some cases

1. ent do not support migration default, but it's ok if you defined any default - phase 1 - this pr
2. ent support default migration, but you have to define the default like the db return - phase 2
3. ent can be more smart to support default migration, understand `'val'::text` is the same as `val` - phase 3

Reply:
@a8m ignore error is ok to me, I try to not change too much, so, keep the error like this.

Reply:
Hey @wenerme 👋 

> how about just track the raw default without the Scan when read from db ?

I'm OK with this.